<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicTestWindow</class>
 <widget class="QMainWindow" name="GraphicTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphicTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="PhGraphicView" name="graphicView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionInfos"/>
    <addaction name="actionImage_1"/>
    <addaction name="actionImage_2"/>
    <addaction name="actionRectangles"/>
    <addaction name="actionArrows"/>
    <addaction name="actionDisc"/>
    <addaction name="actionCharacters"/>
    <addaction name="actionStatic_text"/>
    <addaction name="actionMoving_text"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionMove_up"/>
    <addaction name="actionMove_right"/>
    <addaction name="actionMove_down"/>
    <addaction name="actionMove_left"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChange_text"/>
    <addaction name="actionChange_font"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDisplay"/>
  </widget>
  <action name="actionChange_font">
   <property name="text">
    <string>Change font...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionInfos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Infos</string>
   </property>
  </action>
  <action name="actionRectangles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rectangles</string>
   </property>
  </action>
  <action name="actionArrows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arrows</string>
   </property>
  </action>
  <action name="actionDisc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disc</string>
   </property>
  </action>
  <action name="actionCharacters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Characters</string>
   </property>
  </action>
  <action name="actionStatic_text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Static text</string>
   </property>
  </action>
  <action name="actionMoving_text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Moving text</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>J</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
  <action name="actionMove_up">
   <property name="text">
    <string>Move up</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionMove_right">
   <property name="text">
    <string>Move right</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionMove_down">
   <property name="text">
    <string>Move down</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionMove_left">
   <property name="text">
    <string>Move left</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionChange_text">
   <property name="text">
    <string>Change text...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionImage_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image 2</string>
   </property>
  </action>
  <action name="actionImage_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image 1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhGraphicView</class>
   <extends>QWidget</extends>
   <header>PhGraphic/PhGraphicView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
